plugins/oidc-login.yaml

```diff
@@ -3,30 +3,27 @@ kind: Plugin
 metadata:
   name: oidc-login
 spec:
-  shortDescription: Login for OpenID Connect authentication
+  homepage: https://github.com/int128/kubelogin
+  shortDescription: Log in to the OpenID Connect provider
   description: |
-    This plugin gets a token from the OIDC provider and writes it to the kubeconfig.
-
-    Just run:
-      % kubectl oidc-login
-
-    It opens the browser and you can log in to the provider.
-    After authentication, it gets an ID token and refresh token and writes them to the kubeconfig.
+    This is a kubectl plugin for Kubernetes OpenID Connect (OIDC) authentication.
+    (Credential Plugin Mode)
+      kubectl executes oidc-login before calling the Kubernetes APIs.
+      oidc-login automatically opens the browser and you can log in to the provider.
+      After authentication, kubectl gets the token from oidc-login and you can access to the cluster.
```

Comment:
Can you also link to the documentation with anchor tags, like https://github.com/int128/kubelogin#credential-plugin-mode ?